pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -198,9 +198,9 @@ func (h *NetworkUtilsHandler) SetRandomMac(iface string) (net.HardwareAddr, erro
 		if changed {
 			mac, err = Handler.GetMacDetails(iface)
 			if err != nil {
-				log.Log.Reason(err).Errorf("updated MAC for interface: %s - %s -> %s", iface, currentMac, mac)
 				return nil, err
 			}
+			log.Log.V(4).Infof("updated MAC for interface: %s - %s -> %s", iface, currentMac, mac)
```

Comment:
V(4) is above the default log level. Is that intended?

Reply:
No just a copy paste from below. I've now checked `docs/logging.md` and looks like I should avoid `V(*)`?

Reply:
You can set it to something likte 4 or above it is mostly meant for debugging and would pollute the logs otherwise. Just don't set it if you want that people see it by default. Every kubevirt component has a `-v` flag which defaults to `3`. The same like in kubernetes.